src/layout/MasterLayout.cpp

```diff
@@ -153,28 +153,28 @@ void CHyprMasterLayout::onWindowRemovedTiling(CWindow* pWindow) {
     if (!PNODE)
         return;
 
+    const auto  WORKSPACEID = PNODE->workspaceID;
+    const auto  MASTERSLEFT = getMastersOnWorkspace(WORKSPACEID);
+    const auto* SMALLSPLIT  = &g_pConfigManager->getConfigValuePtr("master:allow_small_split")->intValue;
```

Comment:
missing `static`

Reply:
done.

Reply:
Actually done now, my apologies for wasting your time on the last push